src/components/ModalBody/MovieModal.tsx

```diff
@@ -1,22 +1,22 @@
-import React, { useContext } from "react";
+import React from "react";
 import { LabeledInput } from "../Forms/LabeledInput";
 import { CustomSelect } from "../Forms/CustomSelect";
 import { Button } from "../Button";
-import { DialogContext, actionControlVisibility } from "../../context/DialogContext";
 import { CustomCheckbox } from "../Forms/CustomCheckbox";
+import { useDispatch } from "react-redux";
+import { toggleModal } from "../../store/reducers/appReducer";
 
 export const MovieModal:React.FunctionComponent = () => {
+    const dispatch = useDispatch()
     const resetHandler = () => {
```

Comment:
I would say, implementing that logic can be put inside appropriate reducer to avoid repeating yourself.